core/templates/services/i18n-language-code.service.ts

```diff
@@ -204,6 +204,14 @@ export class I18nLanguageCodeService {
   get onPreferredLanguageCodesLoaded(): EventEmitter<string[]> {
     return this._preferredLanguageCodesLoadedEventEmitter;
   }
+
+  isClassroomnNameTranslationAvailable(classroomName: string): boolean {
```

Comment:
Are there other places in the code where you could use this function? If yes, let's use it. If not, then I was  wrong, and I think it's not worth adding this as a function, let's revert, sorry.

Reply:
Yes it is used here https://github.com/oppia/oppia/blob/e1d383fbb6c0e71b4a972922b27abd89a5265005/core/templates/pages/topic-viewer-page/stories-list/topic-viewer-stories-list.component.ts#L97

Once M2.2 PR is merged I will remove it.